PayPal.xcodeproj/project.pbxproj

```diff
@@ -1365,7 +1364,6 @@
 				80E2FDBE2A83528B0045593D /* CheckoutOrdersAPI.swift in Sources */,
 				3B79E4F72A8503CA00C01D06 /* UpdateVaultVariables.swift in Sources */,
 				8048D28C270B9DE00072214A /* ConfirmPaymentSourceResponse.swift in Sources */,
-				3B109B3D2A85D1CA00D8135F /* MockGraphQLClient.swift in Sources */,
 				3D1763A22720722A00652E1C /* CardResult.swift in Sources */,
```

Comment:
Hey Steven, why are these diffs showing here?

Reply:
That showed up in [this PR](https://github.com/paypal/paypal-ios/pull/205/files) too. It must have been forgotten in the original PR to remove that file, and Xcode is catching that fix now (could be also due to Xcode 15 bump).

I think it's fine to let slip in one of these PRs